operatorcourier/cli.py

```diff
@@ -32,17 +38,16 @@ def parse(self):
         args = parser.parse_args(sys.argv[1:2])
         if not hasattr(self, args.command):
             parser.error('Unrecognized command')
```

Comment:
why remove whitespace here? this seems like noise.

Reply:
I'll remove those then. I think my IDE did this, but I'll make sure this doesn't happen in the future.